### PR TITLE
Fix: Donors list table should account for sub-directory installs' 

### PR DESCRIPTION
**Pull request by Genevieve-K**
Feedback: https://feedback.givewp.com/bug-reports/p/donors-list-table-should-account-for-sub-directory-installs

There was the same problem for donations, and it was quickly fixed. One year ago, I reported the problem for donors. I suggest the same correction to set window.location.

### CODE REVIEW

File: src/Donors/DonorsAdminPage.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ function showReactTable() {
                     },
                 })
                     .then((res) => {
-                        window.location.reload();
+                        window.location.href = '/wp-admin/edit.php?post_type=give_forms&page=give-donors';
```
Comment (kjohnson) [inline src/Donors/DonorsAdminPage.php:138]:
```suggestion
                        window.location.href = '<?php echo admin_url('edit.php?post_type=give_forms&page=give-donors'); ?>';
```

Comment (kjohnson) [reply src/Donors/DonorsAdminPage.php:138]: Being that this is PHP we can inject the generated admin URL using WordPress functions.

Comment (Genevieve-K) [reply src/Donors/DonorsAdminPage.php:138]: You're right, it's better. We can also do that in DonationForms. I copied the code.

**Comment by Genevieve-K** [comment]
Sorry, this doesn't fix the problem. I tested too quickly. The link for the donor name is correct, not the link for 'edit'. Here is the HTML:
```
<th>
	<span class="interweave">
		<div class="donorInformation">
			<img class="donorInformation__gravatar" src="..." alt="donor name" loading="lazy">
			<a href="**https://localhost/test-give/wp-admin**/edit.php?post_type=give_forms&amp;page=give-donors&amp;view=overview&amp;id=12">_name_</a>
			<address class="donorInformation__email">_email_</address>
		</div>
	</span>
	<div role="group" aria-label="Actions" class="...==">
		<div class="...==">
			<a href="**/wp-admin/edit.php**?post_type=give_forms&amp;page=give-donors&amp;view=overview&amp;id=12" class="...">Edit </a>
			<button type="button" data-actionid="12" class="...">Delete </button>
		</div>
	</div>
</th>
```
I try to understand... To do like for DonationForms...
In Donors/Endpoints/ListDonors.php:

**Comment by kjohnson** [comment]
The difference is that `DonorsRowActions.tsx` is not prepending the edit URL with `window.GiveDonations.adminUrl` (comparing against `DonationRowActions.tsx`).

---

### [Donations/resources/components/DonationRowActions.tsx](https://github.com/impress-org/givewp/blob/develop/src/Donations/resources/components/DonationRowActions.tsx)
```jsx
<RowAction
    href={
        window.GiveDonations.adminUrl +
        `edit.php?post_type=give_forms&page=give-payment-history&view=view-payment-details&id=${item.id}`
    }
    displayText={__('Edit', 'give')}
/>
```

### [Donors/resources/components/DonorsRowActions.tsx](https://github.com/impress-org/givewp/blob/develop/src/Donors/resources/components/DonorsRowActions.tsx)
```jsx
<RowAction
    className={styles.action}
    href={`/wp-admin/edit.php?post_type=give_forms&page=give-donors&view=overview&id=${item.id}`}
    displayText={__('Edit', 'give')}
/>
```

**Comment by kjohnson** [comment]
Closing in favor of https://github.com/impress-org/givewp/pull/6999.

@Genevieve-K I'm including a build in the new PR that you can use to test my changes.

**Pull request closed without merging**